pep-0XXX-environment-markers.rst

```diff
@@ -0,0 +1,174 @@
+PEP: 0XXX
+Title: Environment Markers
+Version: $Revision$
+Last-Modified: $Date$
+Author: James Polley <jp@jamezpolley.com>
+Status: Draft
+Type: Informational
+Content-Type: text/x-rst
+Created: 03-Jul-2015
+Post-History: 03-Jul-2015
+Replaces: 0345 (part of)
+
+
+Abstract
+========
+
+An **environment marker** describes a condition about the current execution
+environment. They are used to indicate when certain dependencies are only
+required in particular environments, and to indicate supported platforms
+for distributions with additional constraints beyond the availability of a
+Python runtime.
+
+Environment markers were first specified in PEP-0345[1]. PEP-0426[2] (which
+would replace PEP-0345) proposed extensions to the markers. When
+2.7.10 was released, even these extensions became insufficient due to
+their reliance on simple lexical comparisons, and thus this PEP has
+been born.
+
+
+Rationale
+=========
+
+Many Python packages are written with portability in mind.
+
+For many packages this means they aim to support a wide range of
+Python releases. If they depend on libraries such as argparse or
+simplejson - which started as external libraries, but later got
```

Comment:
unittest2 should be used on all versions of Python - it has improvements from 3.6 backported to everything-else. Using the standard library version on 3.x is a poor idea and just makes using features harder. unittest2 <- the 2 is as in 'greater than 1', not 'for python 2.x'.